src/routes/Profile/WorkHistory/utils.ts

```diff
@@ -17,9 +17,11 @@ export const getWorkHistoryTableRows = (data: Tyopaikka[]): WorkHistoryTableRow[
   data.reduce((rows: WorkHistoryTableRow[], row, key) => {
     const { toimenkuvat } = row;
     const alkuPvm = Math.min(...toimenkuvat.map((t) => new Date(t.alkuPvm).getTime()));
-    const loppuPvm = Math.max(
-      ...toimenkuvat.filter((t) => t.loppuPvm).map((t) => new Date(t.loppuPvm as unknown as string).getTime()),
-    );
+    const loppuPvm = toimenkuvat.some((t) => !t.loppuPvm)
```

Comment:
In this case, the `t` is pretty obvious but still might be nicer to read if it is `toimenkuva`